build.gradle

```diff
@@ -66,7 +66,7 @@ allprojects {
         releaseVersion = 'r2.1.1'
         releaseWebsite = 'https://github.com/google/ExoPlayer'
 
-        muxCoreVersion = "7.0.10"
+        muxCoreVersion = "dev-session-data-event-e5d0aff"
```

Comment:
Do you plan to change this as part of a release? (not sure if this was accidentally committed)

Reply:
I was going to update it before moving this PR out of Draft, and after releasing a MuxCore with the required features.

If r2_10_6 was building, we could still run tests this way.